purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/Constants.kt

```diff
@@ -6,4 +6,6 @@ object Constants {
     const val googlePurchaseToken = "GOOGLE_PURCHASE_TOKEN"
     const val productIdToPurchase = "PRODUCT_ID_TO_PURCHASE"
     const val basePlanIdToPurchase = "BASE_PLAN_ID_TO_PURCHASE"
+    // comma separated list of active entitlements to verify
+    const val activeEntitlementIdsToVerify = "ACTIVE_ENTITLEMENT_IDS_TO_VERIFY"
```

Comment:
This allows to keep sharing the same tests between projects. I thought about separating the integration tests and have a test suite pointing to the load shedder and a suite pointing to production, and we might have to do that at some point as we want to get more comprehensive integration tests. But for now, they can still be shared I think. Lmk if you would prefer me to separate the test suites.

Reply:
Yes I think for now it makes sense to share them for now